src/api/app/views/webui/project/_tabs.html.haml

```diff
@@ -36,14 +36,15 @@
       - if project.staging_project?
         %li.nav-item
           = tab_link('Staging', staging_workflow_path(project.staging_workflow.project), active)
-      - elsif policy(project.staging || Staging::Workflow.new(project: project)).create?
-        %li.nav-item
-          = tab_link('Staging', new_staging_workflow_path(project: project), active)
-      - elsif project.staging.persisted?
-        -# If Staging::Workflow.new(...) is executed above, project.staging will be a new object.
+      - elsif project.staging && project.staging.persisted?
+        -# If Staging::Workflow.new(...) is executed below, project.staging will be a new object.
            It will be persisted if the project is really a staging workflow project.
         %li.nav-item
           = tab_link('Staging', staging_workflow_path(project), active)
+      - elsif policy(project.staging || Staging::Workflow.new(project: project)).create?
```

Comment:
`project.staging`, shouldn't be necessary, right?

Reply:
True

Reply:
removed